ui/layouts/MobileApp.tsx

```diff
@@ -58,18 +64,20 @@ const LogoBox = styled(motion.div)(
   variant({
     variants: {
       big: {
-        width: isCareEnv ? '183px' : '124px',
-        height: '20px',
+        width: logoBigWidth,
+        height: logoBigHeight,
+        maxWidth: '100%',
       },
       small: {
-        width: isCareEnv ? '92px' : '61px',
-        height: '10px',
+        width: logoSmallWidth,
+        height: logoSmallHeight,
+        maxWidth: '100%',
       },
       sticky: {
-        top: 8,
+        top: 4,
         position: 'fixed',
-        width: isCareEnv ? '92px' : '61px',
-        height: '10px',
+        width: logoSmallWidth,
+        height: logoSmallHeight,
```

Comment:
Is it possible to center the logo or make it as wide as the white boxes below?
<img width="439" alt="image" src="https://user-images.githubusercontent.com/1375839/92221516-e788cd80-ee9d-11ea-8e44-b3e18e6590a4.png">

Reply:
Hmm. Good question! Technically it's not that easy. The logo element is also used on '/my-checkins' and is fixed positioned there. For the moment I would like not to change this if not really necessary. But we need a more stable way of displaying logos when we get a general whitelabeling solution.